packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -287,7 +287,11 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   _DragState _state = _DragState.ready;
   late OffsetPair _initialPosition;
   late OffsetPair _pendingDragOffset;
-  late OffsetPair _finalPosition;
+
+  /// The position of the last pointer event received.
```

Comment:
Speaking of documentation… maybe we could expand on this one a bit so it more closely aligns with our [style guidelines](https://github.com/flutter/flutter/blob/05f294c7116f30a93195b479d55539f457edd83f/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-useless-documentation).

Possible things to mention:
- It's stored as a local/global `OffsetPair`
- It's used to create `DragEndDetails`

Reply:
How about

```dart
/// The local and global offsets of the last pointer event received.
///
/// It is used to create the [DragEndDetails], which provides information about
/// the end of a drag gesture.
OffsetPair get lastPosition => _lastPosition;
```

Is there anything you think it could be improved?

Reply:
Nope, everything LGTM after this change!